tiledb/api/src/array/attribute/mod.rs

```diff
@@ -473,330 +476,18 @@ impl Builder {
     }
 }
 
-/// Trait for data which can be used as a fill value for an attribute.
-pub trait IntoFillValue {
```

Comment:
This was moved into `tiledb-common` because of the impls which build upon `PhysicalType`.   I did this before adding the `CellValue` trait; that would probably be more appropriate to use and we could move this trait back into `tiledb-api`.